### PR TITLE
Adds Guarana soda and Guarana vending machine.

### PR DESCRIPTION
**Pull request by SrPiwin**
## About The Pull Request
Added a new soda called Guarana.
image

## Why It's Good For The Game
It's a new soda that players recognize.

## Changelog
:cl:
add: Added a new soda called Guarana Antarctica
/:cl:

### CODE REVIEW

File: panelastation/code/modules/vending/cola.dm [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/obj/machinery/vending/cola/guarana
+	icon = 'panelastation/icons/obj/vending.dmi'
+	icon_state = "Guarana_Machine"
+	name = "\improper Guaran� Antarctica Vendor"
+	desc = "The flavor of Brazil in space!"
+	product_slogans = "Este � o sabor!"
```
Comment (Nopm) [inline panelastation/code/modules/vending/cola.dm]: No non-english strings are allowed in the code.

**Review by Nopm — CHANGES_REQUESTED** [review]

File: panelastation/code/modules/vending/cola.dm [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/obj/machinery/vending/cola/guarana
+	icon = 'panelastation/icons/obj/vending.dmi'
+	icon_state = "Guarana_Machine"
+	name = "\improper Guaraná Antarctica Vendor"
+	desc = "The flavor of Brazil in space!"
```
Comment (Nopm) [inline panelastation/code/modules/vending/cola.dm]: Why did you add an entirely new vending machine just for one drink? Remove it.

**Review by Nopm — CHANGES_REQUESTED** [review]

File: panelastation/code/modules/vending/cola.dm (hunk 2 above)
Comment (Nopm) [reply panelastation/code/modules/vending/cola.dm]: Also fix Travis

Comment (SrPiwin) [reply panelastation/code/modules/vending/cola.dm]: It's not only for a drink, it's just a re-skin of the cola vending machines.

Comment (Nopm) [reply panelastation/code/modules/vending/cola.dm]: It's not a reskin, there is now a completely different vending machine in the code for no good reason.

Comment (SrPiwin) [reply panelastation/code/modules/vending/cola.dm]: Alright

**Review by Nopm — APPROVED** [review]

**Pull request merged**